provision/acc_provision/templates/provision-config.yaml

```diff
@@ -61,3 +61,6 @@ net_config:
 registry:
   image_prefix: noiro                   # e.g: registry.example.com/noiro
   # image_pull_secret: secret_name      # (if needed)
+
+kube_config:
+  ovs_memory_limit: "10Gi"       #e.g:  memory: "10Gi"
```

Comment:
Hate to nit pick, but we seem to be following the convention of commenting out any configuration which is optional for the user to set, and leave only that configuration uncommented which the user absolutely needs to set/override. So my suggestion would be to say:
  # ovs_memory_limit: "10Gi"       # Override if needed, default is 10Gi